### PR TITLE
OpenFGA Project security self-assessment update

### PR DESCRIPTION
**Pull request by lj365**
Basic Threat Landscape
Secure Development Practices improved with status badges
all-around fixes

### CODE REVIEW

File: assessments/projects/openfga/self-assessment.md
Comment (JustinCappos) [inline assessments/projects/openfga/self-assessment.md:3]: Would it be okay to change this so it acknowledges the Security Pals effort, but indicates the maintainers have completed the process?

**Review by JustinCappos — APPROVED** [review]
I'm fine overall with the changes, but would appreciate the Security Pals being acknowledged.  

Please also open an issue if you're ready for us to proceed with a joint assessment.

File: assessments/projects/openfga/self-assessment.md
Comment (lj365) [reply assessments/projects/openfga/self-assessment.md:3]: Totally.

File: assessments/projects/openfga/self-assessment.md
Comment (rhamzeh) [inline assessments/projects/openfga/self-assessment.md:3]: Maybe this?
```suggestion
This assessment was contributed to by community members as part of the [Security Pals](https://github.com/cncf/tag-security/issues/1102) process as well as the core maintainer team. 

```

File: assessments/projects/openfga/self-assessment.md
Comment (rhamzeh) [inline assessments/projects/openfga/self-assessment.md:282]:
```suggestion
    - Describe the authorization model with [types], [relations] and [conditions] 

```

File: assessments/projects/openfga/self-assessment.md
Comment (rhamzeh) [inline assessments/projects/openfga/self-assessment.md:280]:
```suggestion
  openfga.language:

```

**Comment by JustinCappos** [comment]
Can you fix the issues causing the linter to fail?  I think this is ready to merge after....

**Comment by lj365** [comment]
> Can you fix the issues causing the linter to fail? I think this is ready to merge after....

Yes will do. Lots of these were not caught on previous pull requests, silently failing as it appears. 
https://github.com/cncf/tag-security/actions/runs/7575106912

The spelling checks are quite constraining to describe stuff technically

**Comment by JustinCappos** [comment]
> > Can you fix the issues causing the linter to fail? I think this is ready to merge after....
> 
> Yes will do. Lots of these were not caught on previous pull requests, silently failing as it appears. Especially, the line-length > 80 https://github.com/cncf/tag-security/actions/runs/7575106912
> 
> The spelling checks are quite constraining to describe stuff technically as well

For anything like the spell checker missing words, please just fix the actually wrong items and let me know.  I'll either add the words to the spell checker or we will just skip that check...

**Comment by lj365** [comment]
> > > Can you fix the issues causing the linter to fail? I think this is ready to merge after....
> > 
> > 
> > Yes will do. Lots of these were not caught on previous pull requests, silently failing as it appears. Especially, the line-length > 80 https://github.com/cncf/tag-security/actions/runs/7575106912
> > The spelling checks are quite constraining to describe stuff technically as well
> 
> For anything like the spell checker missing words, please just fix the actually wrong items and let me know. I'll either add the words to the spell checker or we will just skip that check...

done that.

**Comment by JustinCappos** [comment]
Okay, thanks for the changes.  I checked with the chairs and it is okay to change the linter to make this pass.  I'll squash and merge this!

**Pull request merged**
